librocksdb_sys/Cargo.toml

```diff
@@ -10,7 +10,7 @@ bzip2-sys = { version = "0.1.11+1.0.8", features = ["static"] }
 libc = "0.2.11"
 libtitan_sys = { path = "libtitan_sys" }
 libz-sys = { version = "1.1", features = ["static"] }
-openssl-sys = { version = "0.9.54", optional = true, features = ["vendored"] }
+openssl-sys = { version = "0.9.54", optional = true }
```

Comment:
what's `vendored` features used for?

Reply:
To build and statically link openssl library.